dropwizard-auth/src/main/java/io/dropwizard/auth/Authorizer.java

```diff
@@ -17,4 +18,16 @@
      * @return {@code true}, if the access is granted, {@code false otherwise}
      */
     boolean authorize(P principal, String role);
```

Comment:
This method is called by the Filter, why not delete or at least deprecate it?

Reply:
Done, at least in the code.
Should I mention this in the release notes and/or elsewhere?

Reply:
@cyberdelia Good catch! Please add it to the release notes in `./docs/source/about/release-notes.rst`.

Reply:
Done!